html/.gitignore

```diff
@@ -1,4 +1,3 @@
 dist/
 es5/
 storybook-static/
-.nyc_output/
```

Comment:
do we not have a generated nyc_output dir anymore?

Reply:
we do not

Reply:
I just checked and we do still have one. If you run `npm run coverage` you can see it. So this cannot be removed. 
![Screen Shot 2020-01-29 at 9 00 30 AM](https://user-images.githubusercontent.com/2117593/73362983-d6c40980-4275-11ea-9d9a-a970a87aff0a.png)